### ISSUE TITLE
CSO Dashboard

### ISSUE THREAD

**Issue by evrimm** [unicef/etools-issues#415]

**Comment by marko911** [comment]
@ntrncic Need endpoint for this table.

**Comment by evrimm** [comment]
@marko911 Can the % disburs. to date number round to one decimal place (e.g 40.7%)?

**Comment by marko911** [comment]
@evrimm yes of course. How do you want me to notify of change, new waffle card or just comment here ? @anusenan

**Comment by evrimm** [comment]
@marko911 comment here is sufficient

**Comment by marko911** [comment]
@evrimm changes have been made and will be pushed at next staging push
